bash/solutii/marius.spinu/Up/up.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+#Author:Spinu Marius
+#Description: Simulating a `return to parents directory` of a directory
+
+Up ()
+{
+    for ((i=1;i<="$1";i++)); do
+        cd ../
+    done
+    exec /bin/bash
```

Comment:
I don't want to spawn a new bash process. I want my current bash process to have the directory changed.